Lib/test/test_pdb.py

```diff
@@ -2933,8 +2933,28 @@ def test_pdbrc_basic(self):
         """)
 
         stdout, stderr = self.run_pdb_script(script, 'q\n', pdbrc=pdbrc, remove_home=True)
+        self.assertNotIn("SyntaxError", stdout)
         self.assertIn("a+8=9", stdout)
 
+    def test_pdbrc_empty_line(self):
+        """Test that empty lines in .pdbrc are ignored."""
+
+        script = textwrap.dedent("""
+            a = 1
+            b = 2
+            c = 3
+        """)
+
+        # The empty lines need to have the same indentation as the other lines
+        pdbrc = textwrap.dedent("""
+            n
+
+        """)
```

Comment:
Should we have a test for comments as well?

Reply:
We did have a test for comments if you take a look above, where I added an extra check. I had the thought to test comments in pdbrc, I just did not check the result. Comments generates a SyntaxError but it does not affect other commands. So I just added the SyntexError check in the existing test.